docs/API.md

```diff
@@ -435,6 +435,8 @@ Options can be configured as a parameter to the [init()](./interfaces/tracer.htm
 | logInjection    | `DD_LOGS_INJECTION`                | `false`        | Enable automatic injection of trace IDs in logs for supported logging libraries. |
 | tags            | `DD_TAGS`                          | `{}`           | Set global tags that should be applied to all spans and metrics. When passed as an environment variable, the format is `key:value,key:value` |
 | sampleRate      | `DD_TRACE_SAMPLE_RATE`             | -              | Controls the ingestion sample rate (between 0 and 1) between the agent and the backend. Defaults to deferring the decision to the agent. |
+| rateLimit       | `DD_TRACE_RATE_LIMIT`              | -              | Global rate limit that is applied on the global sample rate and all rules. |
```

Comment:
The old description was more descriptive about what this actually does and the default. I'd say to merge both since the new description added a mention of sampling rules.